### PR TITLE
add cooking skill and rogue's den fire scene

### PR DESCRIPTION
**Pull request by EthanVieira**
* Started the Cooking skill
* Added Rogue's Den Cooking scene
* Added cooked/burnt shrimps/anchovies
* Added The Rogue's Den song
* Added Fishing/Cooking level up sounds
* Added cooking test data
* Modified anchovy fishing required level to 15

image

### CODE REVIEW

**Review by CurtisGreen — COMMENTED** [review]
Minimap is off on the bottom

File: src/items/fish/raw-anchovies.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ export default class RawAnchovies extends RawFish {
 
     // Other
     cost = 15;
+    cookingXp = 30;
+    cookingLvl = 1;
+    burnLvlLimit = 34;
```
Comment (CurtisGreen) [inline src/items/fish/raw-anchovies.js:12]: Didn't know that's how they did it 🤔

File: src/targets/cooking-fire.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            if (pointer.rightButtonDown() && !pointer.leftButtonDown()) {
+                this.createRightClickMenu(pointer.x, pointer.y, this.actions);
+            } else {
+                this.clickTarget();
+            }
+        });
+
+        // Potential TODO: do we want to be able to cook already cooked food to make them burnt?
+        this.validMaterials.add("Raw Shrimps");
+        this.validMaterials.add("Raw Anchovies");
```
Comment (CurtisGreen) [inline src/targets/cooking-fire.js]: I think these can just be done above in the set initialization above unless there's some reason not to

File: src/items/fish/raw-anchovies.js (hunk 1 above)
Comment (EthanVieira) [reply src/items/fish/raw-anchovies.js:12]: I'm not sure if my equation is right, but there is definitely a limit

**Comment by EthanVieira** [comment]
[Cooking](https://trello.com/c/SkuN8uw7/172-cooking)

**Review by CurtisGreen — APPROVED** [review]

**Pull request merged**